packages/jest-editor-support/src/__tests__/SnapshotSupport-test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const path = require('path');
+const SnapshotSupport = require('../SnapshotSupport');
+
+const snapshotFixturePath = path.resolve(__dirname, 'fixtures/snapshots');
+
+test('snapshot-test.example', () => {
+  const filePath = path.join(snapshotFixturePath, 'snapshot-test.example');
+  const required = require(
+    path.join(
+      snapshotFixturePath,
+      '__snapshots__',
+      'snapshot-test.example.snap'
+    )
+  );
+  const results = SnapshotSupport.getMetadataFromTestFile(filePath);
+  Object.keys(required).forEach(snapshot => {
+    const result = results.find(result => result.exportName === snapshot);
+    expect(result).not.toBe(null);
+    expect(result.content).toBe(required[snapshot]);
+  });
+
+  // -> expect(extractSummary(info).summary).toMatchSnapshot();
+  expect(results[0].node.loc.end.line).toBe(111);
+  expect(results[0].node.loc.end.column).toBe(56);
```

Comment:
Can we add an object on top of the test with metadata for all of the snapshots in the file and then test for all of them rather then just asserting that the first one is correct?